hotsos/core/plugins/kernel/sysfs.py

```diff
@@ -16,37 +16,41 @@ def get(relpath):
         @param relpath: path relative to <data_root>sys
         """
         path = os.path.join(HotSOSConfig.data_root, 'sys', relpath)
-        if not os.path.exists(path):
-            return
+        if os.path.exists(path):
+            with open(path) as fd:
+                value = fd.read()
 
-        with open(path) as fd:
-            value = fd.read()
+            return value.strip()
```

Comment:
Better to put this return "inside" `with` ?

Reply:
fixed